features/sql-interactive.feature

```diff
@@ -13,7 +13,7 @@ Feature: Upcasing SQL as I type in SQL interactive mode
     Given I type "select count(*) 'select' -- select "
     Then I should see "SELECT COUNT(*) 'select' -- select "
 
-  Scenario: Typing newlines should upcase strings
+  Scenario: Typing newlines in the prompt should upcase strings
     When I type "select"
-    When I type a newline
+    When I press "RET" with missing buffer error ignored
```

Comment:
I couldn't think of one. The problem is that comint-mode expects to be running  a process, which obviously we don't want to do in tests so I mocked out the bits that start the process. But that causes `comint-send-input` to error so we have to allow the error or the test fails. 

[espuds](https://github.com/ecukes/espuds) could probably do with a `Then` for expecting an error if you're interested :wink: (this might be very hard, I have no idea).